types/bluebird/index.d.ts

```diff
@@ -951,10 +950,8 @@ declare namespace Bluebird {
   export class Disposer<R> {
   }
 
-  export interface Thenable<R> {
-    then<U>(onFulfilled: (value: R) => U | Thenable<U>, onRejected?: (error: any) => U | Thenable<U>): Thenable<U>;
-    then<U>(onFulfilled: (value: R) => U | Thenable<U>, onRejected?: (error: any) => void | Thenable<void>): Thenable<U>;
-  }
+  /** @deprecated Use PromiseLike<T> directly. */
+  export type Thenable<T> = PromiseLike<T>;
```

Comment:
If you intend to deprecate `Bluebird.Thenable`, wouldn't it be better if we replaced all uses of it inside the `index.d.ts` as well?
In general I'm in favor of using `PromiseLike`, since that guarantees future compatibility with the native promise signature, just like the actual bluebird project has a guarantee for compatbility with all native promise implementations.